Changed the title to include SASS

rolazar opened the pull request:
SASS is a very useful tool and makes CSS more readable. SASS allows nesting syntax which groups selectors together in an intuitive way, and SASS allows variables which are useful for having a single source of truth.

If anything should be taught about SASS, it should be nesting syntax and variables. If anything should be taught about Tailwind, it should be utility classes.

Doing a job search, there are very few jobs that require Tailwind, but a large number that require SASS. It makes sense to spend more time teaching SASS because more jobs require it.

rolazar committed:
Changed the title to include SASS

SASS is a very useful tool and makes CSS more readable. SASS allows nesting syntax which groups selectors together in an intuitive way, and SASS allows variables which are useful for having a single source of truth.

If anything should be taught about SASS, it should be nesting syntax and variables. If anything should be taught about Tailwind, it should be utility classes.

Doing a job search, there are very few jobs that require Tailwind, but a large number that require SASS. It makes sense to spend more time teaching SASS because more jobs require it.

tm-LBenson commented:
Tailwind replaced Bootstrap in level 2 because Tailwind is one of the top trending styling frameworks since 2019.
![image](https://github.com/user-attachments/assets/0e55c26b-051c-4ce3-b115-5de609be056d)
 
Also, I really enjoy teaching this topic. If you have strong feelings about it, I will switch it, but would you be open to teaching SASS in week 5 of level 3 instead of removing tailwind in level 2?

rolazar commented:
All I did was add SASS to the title. I wasn't sure where to add specific SASS topics. At the very least, I would like to see SASS variables and nesting syntax taught. They are very simple concepts that can be taught in a few minutes.

rolazar committed:
Replaced media queries with SASS

Media queries are better suited for responsive design, which is taught in level 3.

A quick adaptation for various screen sizes would be to use vw or % to size fonts and elements.
